src/lib/parse/CompoundValueNode.h

```diff
@@ -127,6 +126,7 @@ class CompoundValueNode
   void                check(void);
   std::string         finish(void);
   std::string         render(Format format, const std::string& indent);
+  std::string         renderV2(void);
```

Comment:
For me is ok. I mean, to use "toJson" in the "new renders" and let old render disappear along with all the v1 stuff when time comes.